src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/ExcessHet.java

```diff
@@ -49,12 +49,12 @@
     }
 
     @VisibleForTesting
-    Pair<Integer, Double> calculateEH(final VariantContext vc, final GenotypesContext genotypes) {
-        final GenotypeCounts t = GenotypeUtils.computeDiploidGenotypeCounts(vc, genotypes);
+    static Pair<Integer, Double> calculateEH(final VariantContext vc, final GenotypesContext genotypes) {
+        final GenotypeCounts t = GenotypeUtils.computeDiploidGenotypeCounts(vc, genotypes, true);
```

Comment:
Extract a `static final boolean ` constant `ROUND_GENOTYPE_COUNTS` equal to `true`, and use it here for clarity.